### PR TITLE
Add my story of present-tense

### PR DESCRIPTION
**Pull request by dhyey35**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2340)

### CODE REVIEW

**Comment by dhyey35** [comment]
Please review this, @casey-collab

File: english/present-tense/present-tense.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Boy : I upload a new selfie on facebook.
```
Comment (casey-collab) [inline english/present-tense/present-tense.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by dhyey35** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :checkered_flag:

**Pull request merged**